docker/jenkins2-centos/contrib/kube-slave-common.sh

```diff
@@ -145,7 +145,7 @@ function generate_kubernetes_config() {
       <namespace>${PROJECT_NAME}</namespace>
       <jenkinsUrl>http://${JENKINS_SERVICE_HOST}:${JENKINS_SERVICE_PORT}</jenkinsUrl>
       <jenkinsTunnel>${JNLP_HOST}:${JNLP_PORT}</jenkinsTunnel>
-      <credentialsId>1a12dfa4-7fc5-47a7-aa17-cc56572a41c7</credentialsId>
+      <credentialsId>aerogear-digger-jenkins-kubernetes-service-account-credential</credentialsId>
```

Comment:
I wasn't sure if the id of that credential must be in UUID format or not. This was due to I send the PR for an early review without actually verifying that it works.
Now I tried this and can verify that it works.